src/Tools/PrepareTests/MinimizeUtil.cs

```diff
@@ -53,21 +53,18 @@ internal static void Run(string sourceDirectory, string destinationDirectory, bo
         //  2. Hard link all other files into destination directory
         Dictionary<Guid, List<FilePathInfo>> initialWalk()
         {
-            IEnumerable<string> directories = new[] {
-                Path.Combine(sourceDirectory, "eng"),
-            };
-
-            if (!isUnix)
-            {
-                directories = directories.Concat([Path.Combine(sourceDirectory, "artifacts", "VSSetup")]);
-            }
-
             var artifactsDir = Path.Combine(sourceDirectory, "artifacts/bin");
-            directories = directories.Concat(Directory.EnumerateDirectories(artifactsDir, "*.UnitTests"));
-            directories = directories.Concat(Directory.EnumerateDirectories(artifactsDir, "*.IntegrationTests"));
-            directories = directories.Concat(Directory.EnumerateDirectories(artifactsDir, "RunTests"));
+            List<string> directories =
+            [
+                Path.Combine(sourceDirectory, "eng"),
+                Path.Combine(sourceDirectory, "artifacts", "VSSetup"),
+                .. Directory.EnumerateDirectories(artifactsDir, "*.UnitTests"),
+                .. Directory.EnumerateDirectories(artifactsDir, "*.IntegrationTests"),
+                .. Directory.EnumerateDirectories(artifactsDir, "RunTests")
+            ];
```

Comment:
what black magic is this...

Reply:
Just trying to creat a `List<string>` vs a lazy evaluated `IEnumerable<string>`. No functional changes here.